Core/HLE/sceNetAdhoc.cpp

```diff
@@ -2004,21 +2323,22 @@ static int sceNetAdhocPtpConnect(int id, int timeout, int flag) {
 								// Set Connected State
 								socket->state = ADHOC_PTP_STATE_ESTABLISHED;
 
-								uint8_t * pip = (uint8_t *)&peer.sin_addr.s_addr;
-								INFO_LOG(SCENET, "sceNetAdhocPtpConnect[%i:%u]: Established (%u.%u.%u.%u:%u)", id, socket->lport, pip[0], pip[1], pip[2], pip[3], socket->pport);
+								INFO_LOG(SCENET, "sceNetAdhocPtpConnect[%i:%u]: Established (%s:%u)", id, socket->lport, inet_ntoa(peer.sin_addr), socket->pport);
 								
 								// Success
 								return 0;
 							}
 							
 							// Timeout occured
-							return ERROR_NET_ADHOC_TIMEOUT;
+							return ERROR_NET_ADHOC_CONNECTION_REFUSED; // ERROR_NET_ADHOC_TIMEOUT;
```

Comment:
this one looks scary and probably can be regression over internet and real psp using aemu plugin , note this one if some game compability broken or not in high latency

Reply:
As i remembered i took that return code from JPCSP, it seems they only use a few error codes instead of the whole set like we do.

Reply:
we have a whole set of error code but its not precise like real console return so its broke some register often , im trying to increase the precision of the error by messing with the control and especially this api on digimon games , due to aemu code often make the game timeout to reduce latency to trick some games this is need a proper testing from the community that use upnp to play.

Reply:
embarking quest on monster hunter does relying a lot in this api before the game decide to drop a peer.